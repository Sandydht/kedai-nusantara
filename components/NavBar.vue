<template>
  <div id="navBar">
    <nuxt-link to="/">
      Home
    </nuxt-link>
    <nuxt-link to="/favorite">
      Favorite
    </nuxt-link>
  </div>
</template>

<script>
export default {
  name: 'NavBar'
}
</script>
