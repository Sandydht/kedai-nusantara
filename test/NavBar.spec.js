import { shallowMount } from '@vue/test-utils'
import NavBar from '~/components/NavBar.vue'

describe('NavBar.vue', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallowMount(NavBar)
  })

  test('should have child component nuxt-link', () => {
    expect(wrapper.findAll('nuxt-link').length).toEqual(2)
  })

  test('has expected html structure', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})
