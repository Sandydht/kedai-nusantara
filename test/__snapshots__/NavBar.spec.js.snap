// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavBar.vue has expected html structure 1`] = `
"<div id=\\"navBar\\">
  <nuxt-link to=\\"/\\">
    Home
  </nuxt-link>
  <nuxt-link to=\\"/favorite\\">
    Favorite
  </nuxt-link>
</div>"
`;
